src/renderer/src/lib/Music.ts

```diff
@@ -196,7 +164,7 @@ export async function togglePlay(force?: boolean): Promise<void> {
   await play();
 }
 
-export function seek(range: ZeroToOne): void {
+export function seek(range: VolumeRange): void {
```

Comment:
I feel like the type `ZeroToOne` is a little bit more explicit and general than the type `VolumeRange`. Personally when I read `ZeroToOne` I know that it is an interval `<0; 1>` and most likely a number and not an object because `VolumeRange` reminds me of a descendant of the built-in `Range` class